euphony/build.gradle

```diff
@@ -50,6 +50,25 @@ android {
     testOptions {
         unitTests.returnDefaultValues = true
     }
+
+    gradle.buildFinished {
+        def abiArr = ['arm64-v8a', 'armeabi-v7a', 'x86', 'x86_64']
+        def resultStr = ""
+
+        for (abi in abiArr) {
+            def filePath = 'euphony/.cxx/cmake/debug/' + abi + '/android_gradle_build_stdout_targets.txt'
+            File file = new File(filePath)
+            def line;
+            file.withReader { reader ->
+                while ((line = reader.readLine()) != null) {
+                    if (line == '**** Gtest Success ****')
+                        resultStr += '\n> ' + abi + '/android_gradle_build_stdout_targets.txt\n' + line
+                }
+            }
+        }
+        if (resultStr != "")
+            println '\nPrint build_stdout' + resultStr
+    }
```

Comment:
I tried refactoring for your changes. How about this?
gtest success line is lastline of ```android_gradle_build_stdout_targets.txt```, so I removed while statement and got last line of the file.

@designe , As you mentioned in https://github.com/euphony-io/euphony/discussions/98, is it better to get the test result file from inside the adb shell than this way? 

```groovy
    gradle.buildFinished {
        def abiArr = ['arm64-v8a', 'armeabi-v7a', 'x86', 'x86_64']
        def resultStr = "\n"

        for (abi in abiArr) {
            def filePath = "euphony/.cxx/cmake/debug/$abi/android_gradle_build_stdout_targets.txt"
            def lines = new File(filePath).readLines()
            def result = lines.get(lines.size() - 1)
            resultStr += "> $abi/android_gradle_build_stdout_targets.txt\n $result\n"
        }
        println "Print build_stdout $resultStr"
    }
```

Reply:
Fantastic!

Reply:
```android_gradle_build_stdout_targets.txt``` is repeated, so it might be good to extract it as a variable.
@kuro11pow2 , write more awesome scripts(+@designe 's suggestion) than me!  😄 
```groovy
def logfile = "android_gradle_build_stdout_targets.txt"
```

Reply:
> I tried refactoring for your changes. How about this? gtest success line is lastline of `android_gradle_build_stdout_targets.txt`, so I removed while statement and got last line of the file.
> 
> @designe , As you mentioned in #98, is it better to get the test result file from inside the adb shell than this way?
> 
> ```groovy
>     gradle.buildFinished {
>         def abiArr = ['arm64-v8a', 'armeabi-v7a', 'x86', 'x86_64']
>         def resultStr = "\n"
> 
>         for (abi in abiArr) {
>             def filePath = "euphony/.cxx/cmake/debug/$abi/android_gradle_build_stdout_targets.txt"
>             def lines = new File(filePath).readLines()
>             def result = lines.get(lines.size() - 1)
>             resultStr += "> $abi/android_gradle_build_stdout_targets.txt\n $result\n"
>         }
>         println "Print build_stdout $resultStr"
>     }
> ```

Thanks for your review 👍  I applied your comment to the code. When printing the last line, the following results are printed for abi that are not currently connected. Please let me know your opinion😄 


![image](https://user-images.githubusercontent.com/47289893/136645956-a59e8236-07da-4abd-9d7d-e2cbd2d62796.png)

Reply:
> @yeonns @kuro11pow2 @YoungSeokHong @psh2849 @seoh02h I think if `android_gradle_build_stdout_targets.txt` have the gtest-result, It seems that there is no need to leave additional gtest results file. how do you think about it?

I also think so. We can print out the results in this file.

Reply:
It's great! :)